spec/controllers/messages_controller_spec.rb

```diff
@@ -0,0 +1,110 @@
+require 'rails_helper'
+
+RSpec.describe MessagesController, :type => :controller do
+  describe "Get #show" do
+    let!(:user) { create(:user) }
```

Comment:
The main difference between `let!` and `let` is that `let!` is called in a before block. Here's a good reference: https://www.relishapp.com/rspec/rspec-core/docs/helper-methods/let-and-let